frontend-react/docs/okta.md

```diff
@@ -0,0 +1,11 @@
+# Okta-side configuration
+
+Our frontend is configured to identify as the "Web" application.
+
+## Dev-side configuration
+
+Our use of okta in frontend is configured by the following environment variables whose values can be found in the application listing within Okta:
+-VITE_OKTA_CLIENTID
+-VITE_OKTA_URL
+
```

Comment:
Could we add more documentation around how we get the client id and  OKTA URLs for each environment? 

If that's documented somewhere else can we reference that here? 

Maybe also some details on where these environment variables are used and how the Okta widget is set-up?